prody/dynamics/plotting.py

```diff
@@ -211,7 +211,7 @@ def showProjection(ensemble, modes, *args, **kwargs):
       * 3 modes: :meth:`~mpl_toolkits.mplot3d.Axes3D.plot`"""
 
     import matplotlib.pyplot as plt
-
+    plt.figure()
```

Comment:
There could be some cases people want to plot the projection on top of some other plots. Always creating a new figure for showProjection disallows such manipulations. But I am aware that other plotting functions have this line, so my suggestion is that how about we make a new configuration called for example "new_figure", default to be `True`, and it controls if a new figure is created or not before plotting?